src/app/footer/footer.component.html

```diff
@@ -0,0 +1,68 @@
+<div class="footer bg-footer-lighter pt-5">
+    <div class="container">
+        <div class="row pb-2">
+            <div class="col-md-4 col-sm-6">
+                <div class="widget widget-links widget-light pb-2 mb-4">
+                    <h3 class="widget-title text-light">{{ shopDepartment }}</h3>
+                    <ul class="widget-list">
+                        <li class="widget-list-item" *ngFor= "let shopDepartment of shopDepartments; let i=index;" >
+                            <a class="widget-list-link" id={{i}} href="#">{{ shopDepartment.name }}</a>
```

Comment:
do we need id?

Reply:
@akashsinha13, For reference i kept but doesn't add any value to functionality.